src/utils/storage/index.ts

```diff
@@ -85,13 +88,16 @@ const StorageUtils = {
   lastPromptNotification: storageBuilder(StorageKeysEnum.LastPromptNotification),
   lastSelectedMenu: storageBuilder(StorageKeysEnum.LastSelectedMenu),
   chatInfo: storageBuilder(StorageKeysEnum.chatInfo),
+  blockingStatus: storageBuilder(StorageKeysEnum.BlockingStatus),
+  blockedStatus: storageBuilder(StorageKeysEnum.BlockedStatus),
+  feed: storageBuilder(StorageKeysEnum.Feed),
   clearAll: () => MMKVStorage.clearAll()
 };
 
 const clearAll = () => {
   const onboardingPassword = StorageUtils.onboardingPassword.get();
   MMKVStorage.clearAll();
-  StorageUtils.onboardingPassword.set(onboardingPassword);
+  StorageUtils.onboardingPassword.set(onboardingPassword || '');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is a good one as it ensures that even if `onboardingPassword` is `undefined`, an empty string will be set instead of `undefined`. This can prevent potential issues down the line where `undefined` values might cause problems.
```diff
- StorageUtils.onboardingPassword.set(onboardingPassword);
+ StorageUtils.onboardingPassword.set(onboardingPassword || '');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->